src/transport/AdminPairingTable.h

```diff
@@ -29,8 +29,11 @@ namespace chip {
 namespace Transport {
 
 typedef uint16_t AdminId;
-static constexpr AdminId kUndefinedAdminId = UINT16_MAX;
+typedef uint64_t FabricId;
 
+static constexpr AdminId kUndefinedAdminId   = UINT16_MAX;
+// static constexpr FabricId kUndefinedFabricId = UINT64_MAX;
```

Comment:
Why is this commented-out line here?

Reply:
Because I had realised this was already defined elsewhere so committed it out then forgot to remove